src/util.cc

```diff
@@ -1131,7 +1131,20 @@ double monotonic_now_sec() {
   return (double)tp.tv_sec + (double)tp.tv_nsec / 1e9;
 }
 
-bool running_under_rr() { return getenv("RUNNING_UNDER_RR") != NULL; }
+bool running_under_rr() {
+  static bool rr_under_rr, rr_check_done = false;
+  if (!rr_check_done) {
+    rr_check_done = true;
+    if (getenv("RUNNING_UNDER_RR") != NULL) {
```

Comment:
We might as well remove the `getenv` check, right? We should keep setting the variable, because it's easier for other apps/tools to check.

Reply:
Yep, we can remove it. I just kept it in case you thought it was useful to set this manually occasionally for testing.